desktop/cmp/grid/find/GridFindField.js

```diff
@@ -142,16 +134,6 @@ GridFindField.propTypes = {
     includeFields: PT.arrayOf(PT.string),
 
     /** Names of field(s) to exclude from search. Cannot be used with `includeFields`. */
-    excludeFields: PT.arrayOf(PT.string),
-
-    /**
-     * Field on optional model to which this component should bind its raw (text) value to persist
-     * across renders. Specify this field to control the state of this component directly. These
-     * are both advanced use-cases - this prop is typically left unset.
-     */
-    bind: PT.string,
-
-    /** Optional model for raw value binding - see comments on the `bind` prop for details. */
-    model: PT.object
+    excludeFields: PT.arrayOf(PT.string)
```

Comment:
Are these `bind` and `model` props in use at any client apps? We should probably flag this as a breaking change.